invenio_rdm_migrator/load/postgresql.py

```diff
@@ -57,7 +62,7 @@ def __init__(self, db_uri, table_loads, tmp_dir):
     def _cleanup(self, db=False):
         """Cleanup csv files and DB after load."""
         for table in self.table_loads:
-            table.cleanup_files(db)
+            table.cleanup(db=db)
```

Comment:
`cleanup_files` did not exist 🙄 we might have plenty of this things since we did not have tests...